charger/easee.go

```diff
@@ -386,17 +396,50 @@ func (c *Easee) Enable(enable bool) error {
 		resp.Body.Close()
 	}
 
+	if c.outstandingPauseResumeAck() {
```

Comment:
Es müsste mindestens erstmal geprüft werden, ob hier in die gleiche Richtung enabled wird.

Reply:
Begriffsdefinition: Einen "fail" würde ich nur sehen, wenn die Easee Cloud API gar nicht antwortet, oder eine leere Antwort auf den 202 sendet. Das ist aber ein anderer Fall. Wenn dieser check hier zuschlägt ist ein vorheriger Aufruf korrekt (also 202 mit Tick Info) quittiert worden, wir haben aber in der Zwischenzeit noch keine Bestätigung über den asynchronen Rückkanal erhalten.

> Auch nach gefailtem enable könnte der Anwender doch "off" selektiert haben? Dann willst du ja auch nicht, dass zwischenzeitlich ein "on" doch noch durch käme...

Das können wir aber nicht mehr steuern. Das "on" ist in diesem Fall ja bereits abgesendet und wird (hoffentlich) bei Easee gerade verarbeitet. Das "off" des Users wird warten müssen bis die Antwort von Easee kommt (oder das 30s Timeout zuschlägt).